app/graphql/index.js

```diff
@@ -1,17 +1,20 @@
-const { makeExecutableSchema } = require('graphql-tools'),
-  types = require('./types'),
-  inputs = require('./inputs'),
-  users = require('./users'),
-  healthCheck = require('./healthCheck');
+const { makeExecutableSchema } = require('graphql-tools');
 
-const typeDefs = [types, inputs, ...users.schemas, ...healthCheck.schemas];
+const types = require('./types');
```

Comment:
why did you change this?

Reply:
Amm it's because I like imports to be separated, it doesn't really have any other purpose